locale/en_US/emails.po

```diff
@@ -33,6 +33,22 @@ msgstr ""
 msgid "emails.notification.description"
 msgstr "The email is sent to registered users that have selected to have this type of notification emailed to them."
 
+msgid "discussion.notification.subject"
+msgstr "{$querySubject}"
+
+msgid "discussion.notification.body"
+msgstr ""
+"{$lastContent}<br />\n"
+"<hr />\n"
+"You have a new notification from {$siteTitle}.<br />\n"
+"<br />\n"
+"Do not reply to this mail, please use this link: {$notificationUrl}<br />\n"
+"<br />\n"
+"{$unsubscribeLink}"
```

Comment:
```
{$message}

<hr>
<a href="{$notificationUrl}">Reply to this message</a> from <a href="{$journalUrl}">{$journalName}</a>.
<br><br>
{$subsubscribeLink} 
```

Reply:
@Vitaliy-1 I can't remember how we're handling `{$unsubscribeLink}`. Are these written into the email template or appended automatically to the email template?

Reply:
It was appended automatically for all notification-type emails but not yet handled with a new pattern. I'd prefer to create a new Mailable's trait for this.

Reply:
Ok, so for this PR to be consistent with what's in `main`, should it be in the email template or left out for now?

Reply:
Also, I filed this so it doesn't get forgotten: https://github.com/pkp/pkp-lib/issues/7874

Reply:
Let's leave it out from the email template